cime_config/config_grids.xml

```diff
@@ -4279,6 +4307,11 @@
       <map name="ROF2OCN_LIQ_RMAPNAME">cpl/cpl6/map_r0125_to_oRRS15to5_nn.160720.nc</map>
     </gridmap>
 
+    <gridmap ocn_grid="ICOS10" rof_grid="r0125">
+      <map name="ROF2OCN_ICE_RMAPNAME">cpl/cpl6/map_r0125_to_ICOS10_nco.211212.nc</map>
```

Comment:
@jonbob do you have the smoothed versions of these maps yet? I am a little uncomfortable leaving this as-is with the expectation that we will somehow remember that these maps will be inappropriate for coupled cases in the future.